Documentation/Guides/MainGuide.md

```diff
@@ -0,0 +1,14 @@
+<p align="center">
+    <img src="https://raw.githubusercontent.com/JamitLabs/Jetworking/develop/Logo.png" width=500>
+</p>
+
+`Jetworking` is a **multi-module** iOS and macOS library providing a **user-friendly interface for common networking tasks**.
+
+---
+
+The source code and an example project are hosted [on GitHub](https://github.com/JamitLabs/Jetworking), while all `public` or `open` interfaces of `Jetworking` are documented here. For each module, the documentation also includes a guide with simple sample usage instructions.
+
+The documentation is split into parts representing the main module (`Jetworking`) and the different sub modules:
+
+- [`Jetworking` main module reference](https://jamitlabs.github.io/Jetworking/)
```

Comment:
The `MainGuide.md` is not only used for the primary docs, but also also as the landing page for the sub module docs – therefore I added the the link to the primary documentation. Ideally, we would have a `MainGuide` for each submodule, only linking to all other docs (and not to itself), but this would IMHO cause too much maintenance costs.